.github/workflows/linux.yml

```
@@ -95,12 +97,19 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install clang-7 gfortran libopenmpi-dev python3
-        sudo .github/workflows/dependencies/install_spack
+    - name: Set up Spack
+      uses: spack/setup-spack@v2
+      with:
+        ref: 187b8adb4faa7ad8e5de0baab9a610d158772823
+        buildcache: true
+        color: true
+        path: /opt/spack
```

CI was not happy with an absolute path:
```suggestion
        path: opt/spack
```